estate/__manifest__.py

```diff
@@ -1,16 +1,26 @@
 {
     "name": "Estate",
     "author": "Vauxoo",
+    "category": "Real Estate/Brokerage",
     "depends": [
         "base",
     ],
     "data": [
+        "data/estate.property.type.csv",
         "security/ir.model.access.csv",
+        "security/estate_security.xml",
```

Comment:
This throws an error, if you delete the database and install the module again, ir.model.access.csv will search for groups from estate_security.xml but they don't exist yet
```suggestion
        "security/estate_security.xml",
        "security/ir.model.access.csv",
```